src/dae/daeUtils.cpp

```diff
@@ -169,8 +170,12 @@ static string tmpDir = string(getenv("TMPDIR"));
 string cdom::getRandomFileName() {
     std::string randomSegment;
     // have to createa a buffer in order to make it multi-thread safe
+#ifndef NO_BOOST
     std::string tmpbuffer; tmpbuffer.resize(L_tmpnam*2+1);
     std::string tmp(tmpnam(&tmpbuffer[0]));
+#else
+	std::string tmp(boost::lexical_cast<std::string>(random_generator()()));
+#endif
```

Comment:
would `tmpnam_s(&tmpbuffer[0], L_tmpnam*2+1)` be a better fix for the warning here?

Reply:
We won't know for sure until we actually incorporate it into the viewer and build it.
If the warning is due to a buffer overflow problem, changing it to tmpnam_s may also improve the situation.

Reply:
When I tried to use tmpnam_s, an error appeared and the build failed on Linux.
I don't know why.

~~~
src/dae/daeUtils.cpp: In function ‘std::string cdom::getRandomFileName()’:
src/dae/daeUtils.cpp:174:21: error: ‘tmpnam_s’ was not declared in this scope; did you mean ‘tmpnam_r’?
  174 |     std::string tmp(tmpnam_s(&tmpbuffer[0], L_tmpnam*2+1));
      |                     ^~~~~~~~
      |                     tmpnam_r
~~~

Reply:
i guess I was wrong about the APIs introduced in C11 being available. sorry about that